graphics/macgui/macmenu.cpp

```diff
@@ -1175,6 +1176,14 @@ void MacMenu::renderSubmenu(MacMenuSubMenu *menu, bool recursive) {
 			Common::Rect trect(r->left, y - (_wm->_fontMan->hasBuiltInFonts() ? 1 : 0), r->right, y + _font->getFontHeight());
 
 			_screen.fillRect(trect, _wm->_colorBlack);
+
+			if (_selectedItem != i) {
+				if (menu->items[i]->unicode) {
+					_wm->sayText(menu->items[i]->unicodeText);
+				} else {
+					_wm->sayText(menu->items[i]->text);
```

Comment:
In this case where you are not using a unicode text, it needs to be converted to unicode using the correct encoding, which I suspect is not the default `kUtf8`.
See `MacFontRun::getEncoding()` and `MacMenu::getMenuFont()` which should give you the information you need.